guides/source/active_record_migrations.md

```diff
@@ -976,37 +1209,37 @@ you can manually run `bin/rails db:seed`.
 
 ### Resetting the Database
 
-The `bin/rails db:reset` command will drop the database and set it up again. This is
-functionally equivalent to `bin/rails db:drop db:setup`.
+The `bin/rails db:reset` command will drop the database and set it up again.
+This is functionally equivalent to `bin/rails db:drop db:setup`.
 
 NOTE: This is not the same as running all the migrations. It will only use the
-contents of the current `db/schema.rb` or `db/structure.sql` file.
-If a migration can't be rolled back, `bin/rails db:reset` may not help you. To
-find out more about dumping the schema see [Schema Dumping and You][] section.
+contents of the current `db/schema.rb` or `db/structure.sql` file. If a
+migration can't be rolled back, `bin/rails db:reset` may not help you. To find
+out more about dumping the schema see [Schema Dumping and You][] section.
 
 [Schema Dumping and You]: #schema-dumping-and-you
 
 ### Running Specific Migrations
 
 If you need to run a specific migration up or down, the `db:migrate:up` and
-`db:migrate:down` commands will do that. Just specify the appropriate version and
-the corresponding migration will have its `change`, `up` or `down` method
+`db:migrate:down` commands will do that. Just specify the appropriate version
+and the corresponding migration will have its `change`, `up` or `down` method
 invoked, for example:
 
 ```bash
-$ bin/rails db:migrate:up VERSION=20080906120000
+$ bin/rails db:migrate:up VERSION=20240428000000
 ```
 
 By running this command the `change` method (or the `up` method) will be
-executed for the migration with the version "20080906120000".
+executed for the migration with the version "20240428000000".
 
 First, this command will check whether the migration exists and if it has
 already been performed and will do nothing if so.
```

Comment:
First, this command will check whether the migration exists and if it has
already been performed and if so, it will do nothing.